sdk/go/src/sawtooth_sdk/fake_validator.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
No need to commit a fake validator.  The fake processor works fine against the real validator if the validator is patched and the request is enclosed in a Message.